content/mooreslaw-tutorial.md

```diff
@@ -346,6 +346,8 @@ plt.legend(loc="center left", bbox_to_anchor=(1, 0.5))
 plt.ylabel("# of transistors\nper microprocessor")
 ```
 
+_When run, this produces scatter plot of MOS transistor count per microprocessor every two years with a red line for the ordinary least squares prediction an orange line for Moore's law._
+
```

Comment:
```suggestion
_A scatter plot of MOS transistor count per microprocessor every two years with a red line for the ordinary least squares prediction an orange line for Moore's law._

```
It's a good observation that notebook users won't technically see the output until this is run, but I expect most users will be looking at rendered output (either on the website or having run the cell interactively) so it might be cleaner to leave off the info about how the plot is generated.